### DISCUSSION TITLE
Crash of a Minkowski sum (Leonardo + dodecahedron)

### DISCUSSION THREAD

**Discussion by stla** [CGAL/cgal#6664] [Q&A] [answered]
Hello,

As [you can see](https://github.com/CGAL/cgal/discussions/6658), I successfully did some Minkowski sums. In particular the sum of Leonardo Da Vinci's 72-sided sphere and an octahedron. But the sum of [Leonardo plus a dodecahedron](https://gist.github.com/stla/7536e9e42206c2b38377b1286f0c4169) crashes the R session. Please anybody, could you give a try?

**Comment by pentacular** [comment]
Can you produce a minimal C++ test case that reproduces the issue?

**Comment by stla** [comment]
I already produced one, please click on the link.

I've just checked that *if I triangulate Leonardo* before doing the sum, then there's no crash.

**Reply by pentacular** [reply]
Very nice :)

Next time, could you make the link to the C++ test case a bit more obvious?

I had a look, but I don't see an obvious reason why triangulation of that should matter -- maybe someone else will know.

**Comment by stla** [comment, marked as answer]
Perhaps this is caused by the fact that the four points of the quad faces are not exactly coplanar.

**Reply by pentacular** [reply]
That would make sense.

**Reply by stla** [reply]
Yes, but strange that this works with the octahedron.
